molo/commenting/forms.py

```diff
@@ -40,6 +40,9 @@ def get_comment_object(self):
         CommentModel = self.get_comment_model()
         new = CommentModel(**self.get_comment_create_data())
 
+        if new.parent is None:
+            new = self.check_for_duplicate_comment(new)
+
```

Comment:
You can do the following:
```python
user_model = get_user_model()
try:
    user = user_model.objects.get(username=new.user_name)
    if not user.is_staff:
        new = self.check_for_duplicate_comment(new)
except user_model.DoesNotExist:
    # post_molo_comment may have set the username to 'Anonymous'
    new = self.check_for_duplicate_comment(new)
```

Please update the docstring to say: Overridden to remove dupe comment check **for admins** ...
Please add tests

Refs:
https://docs.djangoproject.com/en/1.9/topics/auth/customizing/#referencing-the-user-model